.goreleaser.prerelease.yml

```diff
@@ -31,6 +32,7 @@ builds:
   main: ./cmd/pulumi
 - id: pulumi-language-nodejs-windows
   binary: pulumi-language-nodejs
+  # gobinary: ../scripts/go-wrapper.sh
```

Comment:
Are the commented out `gobinary` lines still needed?

Reply:
Added a comment.